### PR TITLE
RHDEVDOCS-5667: Updating install build strategies commands

### PR DESCRIPTION
**Pull request by shipsing**
**Purpose**: To resolve this issue: https://issues.redhat.com/browse/RHDEVDOCS-5667

**Aligned team**: DevTools

**Version for cherrypicking**: build-docs-1.0

**Content for preview**: https://66333--docspreview.netlify.app/openshift-builds/latest/installing/installing-sample-build-strategies

**SME review**:  @apoorvajagtap @jkhelil 

**QE review**:  @jitendar-singh 

**Peer review**: @kelbrown20

### CODE REVIEW

File: modules/ob-installing-buildah-strategy.adoc
Comment (kelbrown20) [inline modules/ob-installing-buildah-strategy.adoc:15]: Referenced in [the doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references), we cannot have xrefs in modules. 
You can add the xrefs to a .Additional resources section in a assembly, but they cannot be in modules.

File: modules/ob-installing-buildah-strategy.adoc
Comment (kelbrown20) [inline modules/ob-installing-buildah-strategy.adoc:16]: Same with the xref here

File: modules/ob-installing-s2i-strategy.adoc
Comment (kelbrown20) [inline modules/ob-installing-s2i-strategy.adoc:14]: All xrefs have to be removed from modules.

**Review by kelbrown20 — COMMENTED** [review]
Hello! Just some updates for the xrefs, but otherwise everything else looks good to me!

**Comment by shipsing** [comment]
@kelbrown20, I have incorporated your suggestions. Please verify the same. Thank you.

File: modules/ob-installing-buildah-strategy.adoc
Comment (apoorvajagtap) [inline modules/ob-installing-buildah-strategy.adoc:24]:
```suggestion
$ oc apply -f https://raw.githubusercontent.com/redhat-developer/openshift-builds-catalog/main/clusterBuildStrategy/buildah/buildah.yaml
```

**Review by apoorvajagtap — CHANGES_REQUESTED** [review]

File: modules/ob-about-buildah.adoc
Comment (apoorvajagtap) [inline modules/ob-about-buildah.adoc:9]:
```suggestion
The `buildah` cluster build strategy uses a Dockerfile to build a container image and pushes it to the target registry. You must specify the Dockerfile in your `Build` CR.
```

**Comment by apoorvajagtap** [comment]
LGTM.

File: modules/ob-installing-buildah-strategy.adoc
Comment (adellape) [inline modules/ob-installing-buildah-strategy.adoc:23]: (Here and the other similar references to GitHub URLs)

Technically per our [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites), we are not supposed to include links or source content directly from GitHub without approval from our Support org. Maybe the previous URL being replaced here was approved at some point, but I couldn't verify in the PR or Jira if the new one was. At any rate, I checked w/ kalexander on this one and we're OK to move it along for now (since this is replacing an existing GitHub link anyway), but please consider for future changes. Thank you!

**Review by adellape — COMMENTED** [review]
Not blocking merge for this, but please see 1 comment in-line.

File: modules/ob-installing-buildah-strategy.adoc
Comment (shipsing) [reply modules/ob-installing-buildah-strategy.adoc:23]: @adellape, Thank you for sharing this information. In a future release, plans are to move this entire section of `Installing build strategies` because users need not to install the strategies manually. The Builds Operator would automatically install all these strategies.

Comment (Preeticp) [reply modules/ob-installing-buildah-strategy.adoc:23]: Hey @adellape, the DevTools teams have exceptions to use links from our support team to examples or repos that are maintained by the OpenShift and Developer org in GH.

**Pull request merged**
